src/main/scala/mesosphere/marathon/storage/migration/Migration.scala

```diff
@@ -172,8 +163,8 @@ class Migration(
             case Failure(ex) =>
               throw ex
           }
-        case Some(version) if version == currentBuildVersion =>
-          logger.info("No migration necessary, already at the current version")
+        case Some(version) if version == targetVersion =>
+          logger.info("No migration necessary, already at the target version")
```

Comment:
maybe also print out the version ... ? for debugging

Reply:
Done